### PR TITLE
[#499][#732][#777] Improvement Grafana deployment

### LINKED ISSUE #499: Remove hardcoded Grafana credentials from code

### LINKED ISSUE #732: PV and PVC for Grafana
Need to add PV and PVC for the Grafana deployments.
Currently, we lose dashboars after restarting of a Grafana pod.

### LINKED ISSUE #777: Remove legion package dependencies from enclave-grafana image
We may use [Grafana Provisioning](http://docs.grafana.org/administration/provisioning/).
And in this case we dont need to build our own docker image of Grafana.

### PR DESCRIPTION
**Pull request by mcdoker18**
* Replaced docker image by official one
* Replaced helm deployment by official helm chart
* Added PV and PVC
* Configurable grafana user/password
* Minor fixed

This closes #499 
This closes #732 
This closes #777

### CODE REVIEW

**Comment by kirillmakhonin** [comment]
Maybe we may generate secrets such as Grafana password on deploy? And persist it in Secret for future usage.

**Comment by kirillmakhonin** [comment]
ok to test

**Comment by mcdoker18** [comment]
> Maybe we may generate secrets such as Grafana password on deploy? And persist it in Secret for future usage.

I don't like it. A couple of problems:
* How should we pass the password to the robot test in your case?
* You also must have access to the k8s to use Grafana.
* Out current upgrade strategy is to delete everything in a namespace and create new deployments. The new password will generate again and again.

**Comment by mcdoker18** [comment]
bot, test all

**Review by kirillmakhonin — DISMISSED** [review]

File: deploy/helms/legion-core/templates/metrics/gradana_configmaps.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
```
Comment (botsman) [inline deploy/helms/legion-core/templates/metrics/gradana_configmaps.yaml]: typo in filename: `gradana_configmaps.yaml`

File: deploy/helms/legion/templates/metrics/gradana_configmaps.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
```
Comment (botsman) [inline deploy/helms/legion/templates/metrics/gradana_configmaps.yaml]: same

**Review by botsman — CHANGES_REQUESTED** [review]

**Comment by mcdoker18** [comment]
bot, test all

**Comment by mcdoker18** [comment]
bot, test all

File: Jenkinsfile [hunk 3]
```diff
@@ -337,7 +337,7 @@ EOL
                         }
                     }
                 }
-                
+
```
Comment (alexander-semenets) [inline Jenkinsfile]: extra blank line

Comment (mcdoker18) [reply Jenkinsfile]: Fixed

**Review by kirillmakhonin — APPROVED** [review]

**Review by botsman — APPROVED** [review]

**Review by alexander-semenets — APPROVED** [review]

**Pull request merged**